content/uk/docs/reference/glossary/upstream.md

```diff
@@ -0,0 +1,18 @@
+---
+title: Upstream (розʼяснення)
```

Comment:
Upstream could be the AKA here.

Reply:
In this case, localization for "upstream" is quite problematic in the absence of a corresponding term in Ukrainian. Until we need such duplication in AKA, I would prefer not to use it.